rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModuleWallet.java

```diff
@@ -18,9 +18,12 @@
 
 package co.rsk.rpc.modules.eth;
 
+import org.ethereum.rpc.parameters.HexAddressParam;
+import org.ethereum.rpc.parameters.HexDataParam;
+
 public interface EthModuleWallet {
 
     String[] accounts();
 
-    String sign(String addr, String data);
+    String sign(HexAddressParam addr, HexDataParam data);
```

Comment:
About this change, I think we shouldn't add the specific JsonRPC parameters here, we could keep the string or directly update to the address and data classes. 
The Json PRC parameter classes should be used only under the jsonRPC domain.

Reply:
I understand, I will revert the changes for this one.